dashboards/CCODashboard-Infra/InfraDeploymentGuide.md

```diff
@@ -153,6 +154,21 @@ The following picture shows the steps to configure Azure Security Center plan fo
 4.	Click on **Free** or **Standard** plan and the click **Save**.
 
 ![SecurityCenterStandardRecommendations](/install/images/EnableSecurityCenterStandard.png)
+  
+# Installing the custom connector
+
+The CCO Azure Governance Dashboard requires to install the Power BI Custom Connector located in the same folder as the CCO Governance Dashboard ([CCoDashboardAzureConnector.mez](/dashboards/CCODashboard-Governance/CcoDashboardAzureConnector.mez)). This Custom Connector allows us to leverage information from Azure Management REST APIs that requires POST methods and errors control
+
+To install the custom connector you must copy the file [CCoDashboardAzureConnector.mez](/dashboards/CCODashboard-Governance/CcoDashboardAzureConnector.mez) from the **ccodashboard/dashboards/CCODashboard-Governance/** folder to the folder that Power BI creates by default in the Documents folder in your PC. If this folder doesn't exist, you can create a new one with this name.
```

Comment:
Update the path to CCODashboard-Infrastructure

Reply:
it would be very helpful if the default folder for power bi connectors